libs/wingc/src/dtsify/mod.rs

```diff
@@ -207,9 +207,6 @@ impl<'a> DTSifier<'a> {
 			code.line(format!(
 				"[{TYPE_INTERNAL_NAMESPACE}.INFLIGHT_SYMBOL]?: {inflight_class_name};"
 			));
-			code.line(format!(
-				"_supportedOps(): {TYPE_INTERNAL_NAMESPACE}.OperationsOf<{inflight_class_name}>;"
-			));
```

Comment:
Are we on purpose not adding a `_liftMap` here?

Reply:
I wasn't sure. The tests are still passing after I removed it so maybe it isn't needed?